### ISSUE TITLE
Control the Wind direction and snowflakes dimensions

### ISSUE THREAD

**Issue by DekiGk** [scottschiller/Snowstorm#28]
First of all, thanks for the great plugin.

Is there a way I can control the wind direction? I know that I can control the max and min speed, but I didn`t find a way to control the direction. It seems like it is always random.

Would be nice if there was a way to control the size of the snowflakes other than overwriting the font-size for the snowflakes.

**Comment by ajsnyde** [comment]
I'm pretty sure the wind direction is controlled by your mouse. That is from a demo I was playing with, though.

**Comment by DekiGk** [comment]
Yes, you are right. But, there is a way to turn off this behaviour. Then, on every page reload the wind is random. What I want is the ability to manage from where the wind blows, other than controling it with my mouse :D
